realsense_camera/src/base_nodelet.cpp

```diff
@@ -275,34 +276,67 @@ namespace realsense_camera
   {
     // print list of detected cameras
     std::vector<int> camera_type_index;
-    std::string detected_camera_msg = " - Detected the following cameras:";
+
+    // map camera names to validated firmware
+    std::map<std::string, std::string> camera_name_to_validated_firmware =
```

Comment:
I would not make it constant as we will likely need to change/override these map values if we enable a negative-testing method for QA.